### ISSUE TITLE
System JS to Angular-cli

### ISSUE THREAD

**Issue by vish288** [sathyalog/ng2-ecommerce#5]
Move from systemjs to compose to angular-cli to compose the assets.

**Comment by sathyalog** [comment]
Sure Vish. But prior to this code refactoring, we have to use web services and universal.angular on high priority itseems. Lets discuss in call.

**Comment by vish288** [comment]
yep. all those needs the movement first to enable smooth compose.

**Comment by sathyalog** [comment]
Cool. I haven't worked on angular-cli yet. Could you please help me out in setting environment for it and let me know the process to finish it. Also we need to integrate AOT as well.

**Comment by vish288** [comment]
Github [Repo](https://github.com/angular/angular-cli)
Read about how to setup with Angular-cli [here](https://cli.angular.io/) and [here](https://scotch.io/tutorials/use-the-angular-cli-for-faster-angular-2-projects). 

Start on it and i'll come in if you get stuck anytime.
